.dependabot/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "live"
+    version_requirement_updates: "increase_versions"
+    ignored_updates:
+      # Addons that require ember octane (3.14.0+) can't be updated since we
+      # guarantee support for the latest LTS version of ember. Those can be
+      # removed as soon as a new LTS version of ember is released.
```

Comment:
That's not quite true. `ember-power-select` 3.x requires `ember` 3.11.0+. However in this addon we support the last two LTS version which are 3.12 and 3.8 right now.

Reply:
Corrected the mistake